Formula/openimageio.rb

```diff
@@ -0,0 +1,70 @@
+class Openimageio < Formula
+  desc "Library for reading, processing and writing images"
+  homepage "http://openimageio.org/"
+  url "https://github.com/OpenImageIO/oiio/archive/Release-1.8.7.tar.gz"
+  sha256 "ed23ecdd3e4037a411c69a6a681273d01d4a2fbcc223eac86daba77dd03fc5ed"
+  head "https://github.com/OpenImageIO/oiio.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "boost-python"
+  depends_on "boost-python3"
+  depends_on "ffmpeg"
+  depends_on "freetype"
+  depends_on "giflib"
+  depends_on "ilmbase"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libraw"
+  depends_on "libtiff"
+  depends_on "opencolorio"
+  depends_on "openexr"
+  depends_on "python3"
+  depends_on "webp"
+
+  def install
+    args = std_cmake_args
+    args << "-DSELF_CONTAINED_INSTALL_TREE=ON" # stub from eluded make wrapper
+
+    args << "-DEMBEDPLUGINS=ON" << "-DUSE_OPENGL=OFF" << "-DUSE_QT=OFF" \
+         << "-DUSE_JPEGTURBO=OFF" << "-DUSE_NUKE=OFF" << "-DUSE_FIELD3D=OFF" \
+         << "-DUSE_OPENCV=OFF" << "-DUSE_PTEX=OFF" << "-DUSE_OPENJPEG=OFF"
+
+    mkdir "build-with-python2" do
+      system "cmake", "..", *args
+      system "make", "install"
+    end
+
+    # CMake picks up the system's python dylib, even if we have a brewed one.
+    py3ver = Language::Python.major_minor_version "python3"
+    py3prefix = Formula["python3"].opt_frameworks/"Python.framework/Versions/#{py3ver}"
+
+    ENV["PYTHONPATH"] = lib/"python#{py3ver}/site-packages"
+
+    args << "-DPYTHON_EXECUTABLE='#{py3prefix}/bin/python3'"
```

Comment:
I don't think the single quotes here or anywhere below are needed

Reply:
Done